readthedocs/core/utils/__init__.py

```diff
@@ -132,11 +131,17 @@ def prepare_build(
     options['time_limit'] = int(time_limit * 1.2)
 
     return (
-        update_docs_task.signature(
-            args=(project.pk,),
-            kwargs=kwargs,
-            options=options,
-            immutable=True,
+        chain(
+            update_docs_task.signature(
+                args=(project.pk,),
+                kwargs=kwargs,
+                options=options,
+                immutable=True,
+            ),
+            clean_build_task.signature(
```

Comment:
I think this `chain` should be controlled by a Django setting because this `prepare_build` is used in Corporate as well and I think we don't want to clean artifacts after the build there.

Reply:
Feature flag will be used here. I checked prepare_build in coroporate, no changes are needed there.

Reply:
Do we want to control this at project level?

Reply:
Yep, deactivated by default is in the current discussion

Reply:
I think we are missing the feature flag here.